tests/functional/centos/7/raw-multi-journal/scenario.py

```diff
@@ -0,0 +1,19 @@
+# Basic information about ceph and its configuration
+ceph = {
+    'releases': ['infernalis', 'jewel'],
+    'cluster_name': 'ceph'
+}
+
+# remote nodes to test, with anything specific to them that might be useful for
+# tests to get. Each one of these can get requested as a py.test fixture to
+# validate information.
+nodes = {
+    'mon0': {
+        'username': 'vagrant',
+        'components': ['mon', 'mon_initial_members']
+    },
+    'osd0': {
+        'username': 'vagrant',
+        'components': [],
```

Comment:
you may want to add specific tests for this scenario(s). For collocated journals for example you would need to ensure that the partitions are there on the device, and probably other things as well